train.py

```diff
@@ -469,7 +476,8 @@ def loss_fn(pred, labels):
                     f"{color.yellow}memory: {gpu_mem_stats.max_reserved_gib:5.2f}GiB"
                     f"({gpu_mem_stats.max_reserved_pct:.2f}%)  "
                     f"{color.blue}wps: {round(wps):,}  "
-                    f"{color.magenta}mfu: {mfu:.2f}%{color.reset}"
+                    f"{color.magenta}mfu: {mfu:.2f}%{color.reset} "
+                    f"{color.white}cpu mem(%): {psutil.cpu_percent():.2f}%{color.reset}"
```

Comment:
you mentioned in the PR description that too many small activation offloads would make the program freeze, curious what the cpu_percent looked like over time and whether you expect fixing the number of cpu threads to help here?